src/main/java/frc/robot/utils/DashboardDataContainer.java

```diff
@@ -41,10 +41,10 @@ public DashboardDataContainer() {
         SmartDashboard.putData("Shooter/Enable tuning", new StartEndCommand(shooter::enableTuning, shooter::disableTuning));
 
         // Command groups data
-	    SmartDashboard.putData("CommandGroups/AutoShoot", new AutoShoot(() ->
-		    SmartDashboard.getNumber("Shooter/Shooting Velocity Setpoint", 0)));
-	    SmartDashboard.putData("CommandGroup/CollectCell", new CollectCell());
-	    SmartDashboard.putData("CommandGroup/CollectFromFeeder", new CollectFromFeeder());
+        SmartDashboard.putData("CommandGroups/AutoShoot", new AutoShoot(() ->
+            SmartDashboard.getNumber("Shooter/Shooting Velocity Setpoint", 0)));
+        SmartDashboard.putData("CommandGroup/CollectCell", new CollectCell());
+        SmartDashboard.putData("CommandGroup/CollectFromFeeder", new CollectFromFeeder());
```

Comment:
make new method:
`private double getNumber(String key, double defaultNumber) {
        return SmartDashboard.getNumber(key, defaultNumber)));
}`
**AND**
`private double getNumber(String key) {
        return getNumber(key, 0);
}`

and use it...